Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -56,9 +56,14 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
             var content = new Y.eZ.Content(),
                 version = new Y.eZ.Version(),
                 type = this.get('contentType'),
-                defaultFields = {};
+                defaultFields = {},
+                contentTypeNames,
+                contentTypeName;
 
-            content.set('name', 'New "' + this.get('contentType').get('names')['eng-GB'] + '"');
+            contentTypeNames = type.get('names');
+            contentTypeName = contentTypeNames[this.get('app').get('defaultLanguageCode')] || contentTypeNames[0];
```

Comment:
Indeed, I see how this would be too simplistic. 
So IIUC, replace app attribute with `prioritizedLanguages`, and use first one as default when necessary 
(such as `ez-contentcreateplugin`)

What about the method for retrieving the prioritizedLanguages?
Getting the info from siteaccesses list seems overkill but OTOH it gives a lot of flexibility, not certain if there's any explicit reason not to do this?